docs/reference/vllm.md

```diff
@@ -50,6 +50,50 @@ curl http://127.0.0.1:8000/generate \
         }'
 ```
 
+To start the vLLM OpenAI server use:
+
+```bash
+python3 -m outlines.serve.openai_server
+```
+
+Example of openai_server running on `http://localhost:8000` using local GPTQ quantized model:
+
+```bash
+ python3 -m outlines.serve.openai_server --port 8000 --model ./Mistral-7B-Instruct-v0.2-GPTQ --dtype float16
+```
+
+To use `v1/chat/completions` (or `v1/completions`) to generate an int for exemple use `{"type": "integer"}`:
+
+```bash
+curl http://localhost:8000/v1/chat/completions \
+    -H "Content-Type: application/json" \
+    -d '{
+        "model": "./Mistral-7B-Instruct-v0.2-GPTQ",
+        "messages": [{"role": "user","content": "Hello!"}],
+        "schema": {"type": "integer"},
+        "max_tokens": 5
+        }'
+```
+```bash
+curl http://localhost:8000/v1/completions \
+    -H "Content-Type: application/json" \
+    -d '{
+        "model": "./Mistral-7B-Instruct-v0.2-GPTQ",
+        "prompt": "Hello!",
+        "schema": {"type": "integer"},
+        "max_tokens": 5
+        }'
+```
+To use with Langchain or with OpenAI add the argument `extra_body`:
+
+```bash
+llm = ChatOpenAI( openai_api_key = "EMPTY",
```

Comment:
You would need to add an import here to make it clear that `ChatOpenAI` is not a class implemented in Outlines.